src/theory/bv/rewrites

```diff
@@ -486,6 +525,15 @@
   (not (bvsle x y))
   (bvslt y x))
 
+(define-cond-rule* bv-mult-pow2
+  ((xs ?BitVec :list) (ys ?BitVec :list) (z ?BitVec) (size Int) (n Int))
+  (def
+    (exponent (int.log2 n))
```

Comment:
Also very nitpick but put in one line with the def (as you did below).

Reply:
Do you think it would be a good idea to separate the defs into multiple lines even when there's only one var? I think it makes some sense to visually separate the defs

Reply:
Should be solved now